src/event-helper.js

```diff
@@ -50,11 +50,12 @@ export function createCustomEvent(win, type, detail, opt_eventInit) {
  * @param {string} eventType
  * @param {function(!Event)} listener
  * @param {boolean=} opt_capture
+ * @param {boolean=} opt_passive
```

Comment:
@aghassemi I know you looked into this API - is this ok per your vision?

I'd, TBH, prefer to pass something like `@param {(boolean|ListenerOptions)=} opt_captureOrOptions` and then do:

```
let capture = false, passive = false;
if (opt_captureOrOptions === undefined) {
  // Do nothing.
} else if (typeof opt_captureOrOptions == 'boolean') {
  capture = opt_captureOrOptions;
} else {
  capture = opt_captureOrOptions.capture || false;
  passive = opt_captureOrOptions.passive || false;
}
```

Or something like this. Otherwise, the optional args, all boolean, we'll get eventually confusing.